test/test_images/sender/main.go

```diff
@@ -81,3 +121,8 @@ func writeTerminationMessage(result interface{}) error {
 	}
 	return ioutil.WriteFile("/dev/termination-log", b, 0644)
 }
+
+// isRetryable determines if the err is an error which is retryable
+func isRetryable(err error) bool {
+	return strings.Contains(err.Error(), "404 Not Found") || strings.Contains(err.Error(), "503 Service Unavailable") || strings.Contains(err.Error(), "500 Internal Server Error")
```

Comment:
It should be possible to unwrap the error as an http.Result in order to access the HTTP status code.